docs/manual-EN/5.appendix/gremlin-ngql.md

```diff
@@ -0,0 +1,247 @@
+
+## Example Queries
+
+The examples in this section make extensive use of the toy graph distributed with [Janus Graph](https://janusgraph.org/) called [_The Graphs of Gods_](https://docs.janusgraph.org/#getting-started). This graph is diagrammed below. The abstract data model is known as a [Property Graph Model](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/1.overview/1.concepts/1.data-model.md) and this particular instance describes the relationships between the beings and places of the Roman pantheon.
+
+![image](https://user-images.githubusercontent.com/42762957/71347155-a1f0d280-25a4-11ea-98f0-d1754e9ea3e4.png)
+
+- Insert data
+  
+  ```bash
+  # insert vertex
+  nebula> INSERT VERTEX character(name, age, type) VALUES hash("saturn"):("saturn", 10000, "titan");
+  nebula> INSERT VERTEX character(name, age, type) VALUES hash("jupiter"):("jupiter", 5000, "god");
+
+  gremlin> v1 = g.addV("character").property(T.id, 1).property('name', 'saturn').property('age', 10000).property('type', 'titan');
+  ==>v[1]
+  gremlin> v2 = g.addV("character").property(T.id, 2)property('name','jupiter').property('age', 5000).property('type', 'god');
+  ==>v[2]
+  gremlin> v3 = g.addV("character").property(T.id, 3).property('name',  'prometheus').property('age', 1000).property('type', 'god').next();
+  ==>v[3]
+  gremlin> v4 = g.addV("character").property(T.id, 4).property('name',  'jesus').property('age', 5000).property('type', 'god').next();
+  ==>v[4]
+
+  # insert edge
+  nebula> INSERT EDGE father() VALUES hash("jupiter")->hash("saturn"):();
+  gremlin> g.addE("father").from(v1).to(v2).property(T.id, 3)
+  ==>e[3][1-father->2]
+  ```
+
+- Delete data
+  
+  ```bash
+  nebula> DELETE VERTEX hash("prometheus")
+  gremlin> g.V(v3).drop()
+  ```
+
+- Update vertex
+
+```bash
+nebula> UPDATE VERTEX hash("jesus") SET character.type = 'titan';
+gremlin> g.V(v7).property('age', 6000)
+```
+
+- Fetch data
+  
+  ```bash
+  nebula> FETCH PROP ON character hash("saturn");
+  ===================================================
+  | character.name | character.age | character.type |
+  ===================================================
+  | saturn         | 10000         | titan          |
+  ---------------------------------------------------
+
+  gremlin> g.V(1).valueMap()
+  ==>[name:[saturn],type:[titan],age:[10000]]
+  ```
+
+- Find the name of hercules's grandfather
+
+    ```bash
+    nebula> GO 2 STEPS FROM hash("hercules") OVER father YIELD  $$.character.name
+    =====================
+    | $$.character.name |
+    =====================
+    | saturn            |
+    ---------------------
+
+    gremlin> g.V().hasLabel('character').has('name','hercules').out ('father').out('father').values('name')
+    ==>saturn
+    ```
+
+- Find the name of hercules's father
+
+    ```bash
+    nebula> GO FROM hash("hercules") OVER father YIELD $$.character.name
```

Comment:
;

Reply:
I checked all the statements to add a `;` at the end